ctru-rs/src/services/apt.rs

```diff
@@ -1,25 +1,38 @@
-use std::sync::atomic::{AtomicBool, Ordering};
+use std::lazy::SyncLazy;
+use std::sync::Mutex;
 
-use crate::error::Error;
+use crate::services::ServiceHandler;
 
 #[non_exhaustive]
-pub struct Apt(());
+pub struct Apt {
+    _service_handler: ServiceHandler,
```

Comment:
Most of these services don't need to use `ServiceHandler` because they are already reference counted by libctru:
https://github.com/devkitPro/libctru/blob/ebb53051886cfe966f1aa27ff3dd6689176de483/libctru/source/services/apt.c#L195
https://github.com/devkitPro/libctru/blob/ebb53051886cfe966f1aa27ff3dd6689176de483/libctru/source/services/apt.c#L345

Reply:
But there isn’t an automatic drop, the decrement is still only available in the `aptExit`.

Reply:
We can call the exit functions in our drop like the code in master does. The service will only actually exit once its internal reference count is zero.